docs/connectors/images/index.md

```diff
@@ -0,0 +1,158 @@
+  "pin": true,
+  "links": {
+    "raw": {
+      "use": ":file",
+      "mill": "/blob"
+    },
+    "exif": {
+      "use": "raw",
+      "mill": "/image/exif"
+    },
+    "thumb": {
+      "use": "raw",
+      "pin": true,
+      "mill": "/image/resize",
+      "opts": {
+        "width": "320",
+        "quality": "80"
+      }
+    }
+  }
+}
+```
+
+Here are the major ideas of the schema and how to think about them in your app.
+
+- Input photos will be transformed into 3 files: Raw, EXIF metadata, and thumbnail.
+- Thumbnails will be pinned locally for fast lookup.
+- Raw files will keep the input resolution.
+- Thumbnails will be max 320px wide and have JPEG quality reduction to save storage space.
+
+#### Sharable images
+
+Sharable image threads are optimized for p2p or group sharing of content. Every Textile peer contains a schema (and mill) for handling these kinds of threads, simply named, `media`. You can create a new media thread as follows.
+
+```tab="cmd"
+{{threads.media_add.cmd.code}}
+```
+
+```JavaScript tab="JS HTTP"
+{{threads.media_add.js_http_client.code}}
+```
+
+```JavaScript tab="React Native"
+{{threads.media_add.react_native.code}}
+```
+
+```Swift tab="iOS"
+{{threads.media_add.ios.code}}
+```
+
+```Java tab="Android"
+{{threads.media_add.android.code}}
+```
+
+Like the CameraRoll thread types, Media thread types contain a local file mill that will resample any input image to make it share-ready. Here is what the thread's schema looks like.
+
+```json
+{
+  "name": "media",
+  "pin": true,
+  "links": {
+    "large": {
+      "use": ":file",
+      "mill": "/image/resize",
+      "opts": {
+        "width": "800",
+        "quality": "80"
+      }
+    },
+    "small": {
+      "use": ":file",
+      "mill": "/image/resize",
+      "opts": {
+        "width": "320",
+        "quality": "80"
+      }
+    },
+    "thumb": {
+      "use": "large",
+      "pin": true,
+      "mill": "/image/resize",
+      "opts": {
+        "width": "100",
+        "quality": "80"
+      }
+    }
+  }
+}
+```
+
+Unlike the CameraRoll schema, you can see that images are resampled into a few different variants but the original, full-resolution file is not maintained. You may also notice that the small format of a Media thread is the same format as the thumbnail in the CameraRoll thread. This can be helpful if you add a photo to multiple threads, as there is the opportunity to deduplicate this content.
+
+### Connect to existing photo threads
+
+In many cases, it is just as easy for your app to connect to a user's existing photo threads. Connecting to an existing stream can give your app the ability to provide value across many more photos or simplify photo selection, editing, or sharing workflows. Requesting permission to a user's existing Media or CameraRoll threads is straightforward in your app and is expected to be released to production in the first half of July. Subscribe to this Issue to track our progress on the feature. 
```

Comment:
Missing link

Reply:
nice. using this one https://github.com/textileio/go-textile/issues/694